homeassistant/components/saj/config_flow.py

```diff
@@ -0,0 +1,125 @@
+"""Config flow for saj integration."""
+import logging
+
+import pysaj
+import voluptuous as vol
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_DEVICE_ID,
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_SCAN_INTERVAL,
+    CONF_TYPE,
+    CONF_USERNAME,
+)
+from homeassistant.core import callback
+
+from .const import (  # pylint:disable=unused-import
+    DEFAULT_SCAN_INTERVAL,
+    DOMAIN,
+    ENABLED_SENSORS,
+    INVERTER_TYPES,
+    MAX_SCAN_INTERVAL,
+    MIN_SCAN_INTERVAL,
+)
+from .sensor import CannotConnect, SAJInverter
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for SAJ."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry: ConfigEntry):
```

Comment:
Either provide a full signature or none :)

```suggestion
    def async_get_options_flow(config_entry: ConfigEntry) -> OptionsFlowHandler:
```